manim/mobject/svg/text_mobject.py

```diff
@@ -676,8 +676,45 @@ class MarkupText(SVGMobject):
     r"""Display (non-LaTeX) text rendered using `Pango <https://pango.gnome.org/>`_.
 
     Text objects behave like a :class:`.VGroup`-like iterable of all characters
-    in the given text. In particular, slicing is possible. Text can be formatted
-    using different tags:
+    in the given text. In particular, slicing is possible.
+
+    **What is PangoMarkup?**
+
+    PangoMarkup is a small markup language like html and it helps you avoid using
+    "range of characters" while colouring or styling a piece a Text. You can use
+    this language with :class:`~.MarkupText`.
+
+    A simple example of a marked-up string might be::
```

Comment:
```suggestion
    A simple example of a marked-up string might be:
```

Reply:
This is Rst specific syntax to render the below one as code-block.